shop-service/src/functions/getProductsList/mock.json

```diff
@@ -2,5 +2,5 @@
   "headers": {
     "Content-Type": "application/json"
   },
-  "body": "{\"name\": \"Frederic\"}"
+  "body": "{\"products\": []}"
 }
```

Comment:
Same as previous! Looks like copy and paste :))

Reply:
same as previous - guess it is used for autoswagger

Reply:
> Same as previous! Looks like copy and paste :))

It is a autogenerated code if you start project with serverless Typescript template. It is not polite to offence if you do not know